packages/unlock-js/openapi.yml

```diff
@@ -1554,6 +1554,13 @@ paths:
           description: Recaptcha value to pass.
           schema:
             type: string
+        - in: query
```

Comment:
I _think_ that's the rigth way of adding an optional param... please confirm @searchableguy !